src/css/tabs/pid_tuning.css

```diff
@@ -902,11 +902,16 @@
     display: flex;
     flex-flow: row wrap;
     align-items: flex-start;
-    justify-content: center;
+    justify-content: flex-start;
 }
 
 .tab-pid_tuning .subtab-pid .cf_column {
-    min-width: 600px;
+    flex: 2;
+}
+
+.tab-pid_tuning .subtab-pid .cf_column_right{
+        min-width: 330px;
```

Comment:
Wrong indendation

Reply:
indentation fixed, and fixes for Android done